modules/ROOT/pages/organization.adoc

```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint platform, permissions, configuring, business groups
 
-When you create an Anypoint Platform account, a master (or root) organization is created, and you are assigned as the owner of the organization and automatically inherit the Organization Administrators role. The organization name is whatever name you entered in the Company field in the initial Anypoint Platform signup form. This can be changed in the organization settings.
+When you create an Anypoint Platform account, a master (or root) organization is created, and you are assigned as the owner of the organization. Organization owners automatically inherit the *Organization Administrators* role. 
 
-The organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
+The organization name is the name you entered in the *Company* field in the initial Anypoint Platform signup form. You can change the name in the organization settings.
 
-When you sign into Anypoint Platform, the organization icon and name are displayed on the top right of the page. You can click the organization icon to navigate between business groups. The master organization can contain multiple business groups. You can think of business groups as sub organizations, or children of the master organization.
+An organization is an account where multiple users can share resources, including applications and environments. The level of access users have to various resources depends on their assigned roles and permissions. For example, one user might have permission to manage API alerts, while another user has permission only to view API alerts.
```

Comment:
master org = account, yes? Can we use just one term?

Reply:
Terminology needs to be straightened out for Access Management for sure. But it's going to exceed the purpose of this PR and input from the PM. For now I'm not tackling it.

Reply:
"account" is overloaded between user account and tenant. I'd rather not use account at all if possible.

I agree with this being out of scope of the PR, but my personal perference would be something like:
BG: A node in a tree of (mostly) self-contained resource groups.
Organization: The whole hierarchy of BGs
Master or root BG: the top-level BG that can't be deleted